concent_api/utils/helpers.py

```diff
@@ -159,3 +160,13 @@ def calculate_maximum_download_time(size: int) -> int:
             settings.MINIMUM_UPLOAD_RATE,
         ).total_seconds()
     )
+
+
+def calculate_subtask_verification_time(report_computed_task: message.ReportComputedTask) -> int:
```

Comment:
I am wondering if we could add some unit tests for this function. What it does is basically taking a `datetime.timedelta` object and extracts seconds out of it as `int` value. Not much, but I guess one unit test won't hurt ;)

Or, alternatively, you could change `setUp` in `AuthAcceptOrRejectIntegrationTest`, `AuthGetTaskResultIntegrationTest`, `AcceptOrRejectIntegrationTest`, `GetTaskResultIntegrationTest` to mock `subtask_verification_time` (to return i. e. `timedelta(seconds=X)` instead of mocking `calculate_subtask_verification_time`.

Reply:
:+1: done